domenetjenester/beregning-ytelse/src/main/java/no/nav/ung/sak/ytelse/DagsatsOgUtbetalingsgrad.java

```diff
@@ -0,0 +1,8 @@
+package no.nav.ung.sak.ytelse;
+
+
+import java.math.BigDecimal;
+
+public record DagsatsOgUtbetalingsgrad(BigDecimal dagsats, BigDecimal utbetalingsgrad) {
```

Comment:
Dagsats kan kanskje være Long, slik som i TilkjentYtelsePeriode? Eller brukes denne for mellomregning?

Reply:
Denne er kun for mellomregning. Resultatet i tilkjent ytelse er long